server/db/schema/hostels.schema.ts

```diff
@@ -0,0 +1,45 @@
+import { relations } from 'drizzle-orm';
+import { pgTable, serial, text, timestamp, varchar } from 'drizzle-orm/pg-core';
+
+import { staff } from '.';
+
+export const hostels = pgTable('hostels', {
+  id: serial('id').primaryKey(),
+  address: text('address').notNull(),
+  name: varchar('name', { length: 256 }).notNull(),
+  email: varchar('email', { length: 256 }).notNull(),
+  telephone: varchar('telephone', { length: 13 }).notNull(),
+  alternateTelephone: varchar('alternate_telephone', { length: 13 }),
+  type: varchar('type', {
+    enum: ['boys', 'girls'],
+  }).notNull(),
+  createdAt: timestamp('created_at').defaultNow().notNull(),
+  updatedAt: timestamp('updated_at')
+    .$onUpdate(() => new Date())
+    .notNull(),
+});
+
+export const hostelRelations = relations(hostels, ({ many }) => ({
+  posts: many(hostelPosts),
+}));
+
+export const hostelPosts = pgTable('hostel_posts', {
```

Comment:
Reading that table name I thought it's for any articles that the hostel is releasing.
```suggestion
export const hostelStaff = pgTable('hostel_posts', {
```
Also, hostels have superintendents as well.

Probably look at the board outside the MMCA office or ask Praveen sir directly, he's chill